backend/src/schema/resolvers/searches.js

```diff
@@ -40,8 +39,7 @@ export default {
       YIELD node as resource, score
       MATCH (resource)
       WHERE score >= 0.5
-      AND NOT (resource.deleted = true OR resource.disabled = true
-      OR (:User { id: $thisUserId })-[:BLOCKED]-(resource))
+      AND NOT (resource.deleted = true OR resource.disabled = true)
```

Comment:
I think this introduced a bug: Your search results would contain posts by muted users.

Reply:
this is part of the `UserQuery`, so it would so users who you have blocked... this was specifically asked for.
I have updated the `PostQuery` to remove posts by those you have `Muted`... and found out why it was till passing cypress...